### PR TITLE
Turing Way funding application

### PR DESCRIPTION
**Pull request by KirstieJane**
### Summary

Doesn't fix any open issues (I think).

This is the funding application extension for the next 18 months of _The Turing Way_.

I'm over the word limit in a bunch of sections, so I'm going to keep working, but please let me know what you think!

### List of changes proposed in this PR (pull-request)

* Funding application for Turing Way extension as part of Tools, Practices and Systems programme at Turing Institute

### What should a reviewer concentrate their feedback on?

- [ ] Is anything missing? I feel too close to the project to be able to clearly explain the value 🤦‍♀ so please shout if there are aspects I haven't explained well
- [ ] Are you planning to contribute a chapter? Let me know so I can add you to the list! 💖 
- [ ] Typos and suggestions of where to cut would be great 😩 

### Acknowledging contributors

By submitting this pull request I confirm that:

- [x] all contributors to this pull request who wish to be included are named in [contributors.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/contributors.md).

### CODE REVIEW

**Comment by pherterich** [comment]
Main thoughts: 
line 34 mentions a series of books, will the current "guide to reproducible data science" be one of them? Could you mention other parts of the series here? It comes later in the proposal but I'm wondering if at this stage, it would be useful to say how things fit.

Out of curiosity: is there an easy way of tracking people's first PRs that's not them telling you?

Could the BL be a liaison to the RDA (https://www.rd-alliance.org/) and CODATA (http://www.dcc.ac.uk/about-us/partnerships/codata)? Is that anything you would want to have?

This is technically not a research project, but could you tie it in with your autistica project and claim that this will also provide some kind of insight on how/if/under which conditions people contribute to an open source project? Is there a potential link with Chris's sloan grant?

I'll keep on reading and thinking - there might be more random comics throughout the day.

**Comment by KirstieJane** [comment]
> Main thoughts:
> line 34 mentions a series of books, will the current "guide to reproducible data science" be one of them? Could you mention other parts of the series here? It comes later in the proposal but I'm wondering if at this stage, it would be useful to say how things fit.

I think I cut this for word count reasons.....I'll keep that in mind to see what I can get back in earlier on.

> Out of curiosity: is there an easy way of tracking people's first PRs that's not them telling you?

No, but I had a conversation about building a tool to do this at a hackathon last year.....would be such a cool GitHub plugin!
 
> Could the BL be a liaison to the RDA (https://www.rd-alliance.org/) and CODATA (http://www.dcc.ac.uk/about-us/partnerships/codata)? Is that anything you would want to have?

Yes - definitely, I'm super short on wordcount but I'll try to add these to the community connections section :heart:

> This is technically not a research project, but could you tie it in with your autistica project and claim that this will also provide some kind of insight on how/if/under which conditions people contribute to an open source project? Is there a potential link with Chris's sloan grant?

So, I've kinda tried to keep my projects out of the application so it doesn't seem like a monopoly.....but I'll add the Autistica one in and see whether it survives the word count cull....

> I'll keep on reading and thinking - there might be more random comics throughout the day.

THANK YOU!

File: project_management/tps-funding-application-20190429.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+The book does not currently include the aspects of data science and artificial intelligence that define whether work was ethically designed and reported.
+By curating this knowledge in an accessible format, _The Turing Way_ series will support the The Turing Institute's goals of delivering world class research, training the leaders of tomorrow, and leading the public conversation.
+
+_The Turing Way_ will not replace academic journals or software packages.
+Rather it is a pedagogical opportunity.
+A translation of research outputs into curated tutorials, case studies and checklists.
+The books are Jupyter Books (https://github.com/jupyter/jupyter-book) which build and host static web content directly from markdown files or jupyter notebooks in a GitHub repository.
+They are easy to version control, edit and extend.
+The notebooks are interactive via a Binder plugin.
```
Comment (rainsworth) [inline project_management/tps-funding-application-20190429.md]: Link to Binder?

File: project_management/tps-funding-application-20190429.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+We are committed to giving all early career data scientists the support they need to confidently publish reproducible, accurate, interpretable, and responsible research.
+We will support financial analysts, civil servants, people in third sector organisations, and domain experts who need to evaluate applications of data science and AI.
+_The Turing Way_ books will be accessible to everyone who wants to learn these skills and the content will adapt to accommodate the needs of these communities.
+
+Pathways:
+  * **Training Steering Group** - use of The Turing Way chapters in doctoral training.
+  * **Book dash events** - respond to community requests for content and build confidence in early career researchers to share their skills.
+  * **Research engineering and partnerships teams** - chapter on effectively scoping an interdisciplinary project.
+
+_The Turing Way_ will enable world class research by facilitating communication interdisciplinary teams.
```
Comment (rainsworth) [inline project_management/tps-funding-application-20190429.md]: by facilitating communication *within* interdisciplinary teams?

Comment (KirstieJane) [reply project_management/tps-funding-application-20190429.md]: Yep 🤦‍♀ Thank you!

File: project_management/tps-funding-application-20190429.md (hunk 1 above)
Comment (KirstieJane) [reply project_management/tps-funding-application-20190429.md]: Good call 😸

File: project_management/tps-funding-application-20190429.md [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+Pathways:
+  * **Jupyter and Microsoft Azure development communities** -  contribute and promote _The Turing Way_ across their networks of influence, and to listen to "pain points" in their tools experienced by community members.
+  * **Carpentries community** - co-design training materials and ensure _The Turing Way_ meets the highest pedagogical standards.
+
+### Research challenges
+
+*Max 300 words*
+
+Translating knowledge from one domain to another is always harder than it first appears.
+Tackling the *known unknowns* are the core goal of _The Turing Way_: providing a space for research teams to acquire the skills that they know they lack.
```
Comment (rainsworth) [inline project_management/tps-funding-application-20190429.md]: Tackling the known unknowns *is* the core goal of The Turing Way

Comment (KirstieJane) [reply project_management/tps-funding-application-20190429.md]: ✨ 🙊

**Comment by pherterich** [comment]
"The Turing Way community will support - as co-authors if appropriate - research projects that seek to push back the boundaries of what can be done in transparent, ethical, reproducible and innovative data science."
-> if there are some words left, wondering if you could add something about contributions getting DOIs so they can be counted as outputs on CVs/dissemination plans and thus showcase how a new incentive system could work? Might be too far out, but could be a nice shoutout to early career researchers struggling with culture change...

**Comment by KirstieJane** [comment]
Thank you for the review @rainsworth!! MERGED!!

**Pull request merged**